content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

```diff
@@ -0,0 +1,33 @@
+---
+title: kubeadm reset
+content_type: conceito
+weight: 60
+---
+<!-- overview -->
+Executa uma melhor reversão das alterações feitas pelo `kubeadm init` ou `kubeadm join`.
+
+<!-- body -->
+{{< include "generated/kubeadm_reset.md" >}}
+
+### Redefinindo o fluxo de trabalho {#reset-workflow}
```

Comment:
This title is talking about the workflow for the reset command in the original documentation. This translation is not really delivering it.

Some suggestions:
* "Fluxo do comando `reset`".
* "Fluxo de execução do comando `reset`".